pkgs/development/python-modules/pyeasee/default.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  pytest-asyncio,
+  aioresponses,
+  aiohttp,
+  pysignalr,
+}:
+
+buildPythonPackage rec {
+  pname = "pyeasee";
+  version = "0.8.11";
```

```suggestion
  version = "0.8.11";
  pyproject = true;
```

and set up `build-system = [ setuptools ]`.